Sources/Paywalls/Components/Common/Background.swift

```diff
@@ -69,9 +91,13 @@ public extension PaywallComponent {
 
             case color
             case image
+            case video
 
         }
 
     }
 
 }
+
+public typealias Loop = Bool
+public typealias MuteAudio = Bool
```

Comment:
We probably don't want these to be public? 🤔  Can these be spi-ed if we need these in `RevenueCatUI`?

Reply:
I'll nest it in PaywallComponent.Background. Otherwise I need to make those SPI Internal too.

<img width="945" height="232" alt="Screenshot 2025-10-24 at 1 42 38 PM" src="https://github.com/user-attachments/assets/05023264-7a4d-4f76-97c3-be7b7a2acc30" />